tools/release-prepare.sh

```diff
@@ -66,16 +67,17 @@ for file in "${files_to_change[@]}"; do
     sed -i "s/$prev_ver/$version/" "$file"
 done
 
-# Run `cargo check` to update firecracker and jailer versions in
-# `Cargo.lock`.
-say "Updating lockfile..."
+CHANGED=()
 cargo check
```

Comment:
Do we need this `cargo check`? I think it is picked up by the loop below

Reply:
hmm yeah... where did that come from? fixing